### PR TITLE
Fix plotting of binned data with dtype=datetime64

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

File: python/src/scipp/plotting/model.py [hunk 1]
```diff
@@ -83,6 +80,29 @@ def dims(self, dims):
...
+    def _setup_event_coords(self, data):
+        tmp = data.bins.constituents
+        coord_list = {}
+        array = tmp['data']
+        for dim in data.dims:
+            # Drop any coords not required for resampling to save memory and compute
+            if dim in array.meta:
+                coord = array.meta[dim]
+                if typing.has_datetime_type(coord):
+                    coord = coord - coord.min()
```
Comment (OwenArnold) [inline python/src/scipp/plotting/model.py:99]: Why subtract the min?

**Review by OwenArnold — CHANGES_REQUESTED** [review]
date times on the y-axis require an adjustment in horizontal padding between vertical toolbar and plot. 
image

File: python/src/scipp/plotting/model.py (hunk 1 above)
Comment (SimonHeybrock) [reply python/src/scipp/plotting/model.py:99]: This is how datetime formatting/rebinning is handled, nothing new here, just doing the same that is done for dense coords.

**Comment by SimonHeybrock** [comment]
> date times on the y-axis require an adjustment in horizontal padding between vertical toolbar and plot.
> image

This is unrelated I believe? We have plenty of similar issues in other cases.

File: python/src/scipp/plotting/model.py (hunk 1 above)
Comment (SimonHeybrock) [reply python/src/scipp/plotting/model.py:99]: Basically we cannot rebin datetime, so the min is subtracted such that the integer-valued time-difference can be rebinned instead.

Comment (OwenArnold) [reply python/src/scipp/plotting/model.py:99]: Thanks that makes sense.

**Comment by SimonHeybrock** [comment]
@OwenArnold Did you have more comments on this?

**Comment by OwenArnold** [comment]
> @OwenArnold Did you have more comments on this?

No i'll move the bug regarding the hidden y-axis label to a new/existing issue.

**Review by OwenArnold — APPROVED** [review]

**Pull request merged**